Automattic/jetpack
# Jetpack/Backup/Social: Init new cycles and backport changelogs

#### Changes proposed in this Pull Request:

* Initializes new cycles and backports changes from releasing Jetpack, Backup, and Social.

#### Other information:


#### Jetpack product discussion

n/a

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:

* Proof-read. Make sure versions are correct. The Jetpack changelog and readme should actually be good to go (no need for an edit PR for those) unless you see anything obvious needing a change there. Social and Backup need readme edits so that will come.
